src/site/content/en/blog/airshift-perf-optimization/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "5 ways AirSHIFT made their React app responsive"
```

Comment:
```suggestion
title: Five ways AirSHIFT improved their React app's runtime performance
```

I think we need to avoid the term `responsive`. When you search for `responsive`, the second result is `responsive web design`:

![image](https://user-images.githubusercontent.com/4713486/67997505-9e2e7880-fc11-11e9-8174-9e45db23873c.png)

It's likely that many Search users would search something like `responsive react` and this page might come up in the results. But the post isn't about responsive design. It's about improving user input responsiveness. So I'm going to have to suggest rephrasing our usage of `responsive` throughout this post.

Reply:
SGTM, thanks ;-)